R/topics.R

```diff
@@ -51,21 +52,22 @@ topics_create <- function(name,
                           satisfies_pzs = NULL,
                           message_storage_policy = NULL,
                           schema_settings = NULL,
-                          message_retention_duration = NULL) {
+                          message_retention_duration = NULL,
+                          project = Sys.getenv("GCP_PROJECT")) {
```

Comment:
Switching the base branch for this PR to `dev`. I think this is not needed as I've implemented couple helper methods in #13 to easier switch between GCP_PROJECT_IDs.